spec/controllers/api/v1/inventory_numbers_controller_spec.rb

```diff
@@ -12,17 +13,18 @@
         post :create
       }.to change(InventoryNumber, :count).by(1)
       expect(response.status).to eq(200)
+      expect(parsed_body['inventory_number']).to eql(InventoryNumber.last.code)
     end
   end
 
   describe "PUT /v1/inventory_numbers Delete InventoryNumber" do
     it 'deletes existing inventory_number and returns blank json' do
+      inventory_number
```

Comment:
Hi @steveyken Do we need to add this ? Because in post request we are calling it right ? So i think object will get created. 🤔 

I think this line is added due to use of `let`.

Reply:
Would like to know this as it will be helpful for me. Nothing else 😅

Reply:
Yes I replaced "let!" - I'm not a big fan of it because it's easy to miss. "Let" is much clearer

Reply:
Okay, Sure 👍